tket/tests/test_json.cpp

```diff
@@ -632,6 +632,8 @@ SCENARIO("Test compiler pass serializations") {
   COMPPASSJSONTEST(PlacementPass, gen_placement_pass(place))
   // TKET-1419
   COMPPASSJSONTEST(NoiseAwarePlacement, gen_placement_pass(na_place))
+  COMPPASSJSONTEST(NaivePlacementPass, gen_naive_placement_pass(arc))
+  COMPPASSJSONTEST(LinePlacement, gen_placement_pass(la_place))
```

Comment:
Is the line placement compatible with a check of the NaivePlacementPass, too? Could you add an additional check if that is possible?

Reply:
The addition of the check for `gen_naive_placement_pass`is more papering over an oversight of the last PR. `gen_naive_placement_pass` takes an architecture not a placement object and so they are not compatible